third_party/boringssl/src/METADATA

```diff
@@ -11,8 +11,8 @@ third_party {
     type: GIT
     value: "https://boringssl.googlesource.com/boringssl"
   }
-  version: "7f4f41fa81c03e0f8ef1ab5b3d1d566b5968f107"
-  # from https://chromium.googlesource.com/chromium/src/+/70.0.3538.110/DEPS#141
+  version: "2d98d49cf712ca7dc6f4b23b9c5f5542385d8dbe"
```

Comment:
Is this associated with a boringssl version tag or does chromium not require that it be?

Reply:
It's from a Chromium release, as the comment below certifies.

In general the intent is to use code from stable Chromium releases
because it's been tested and used successfully in the most relevant context.

In particular BoringSSL isn't released separately;
it's primarily a part of Chromium.